### DISCUSSION TITLE
Bounding box jumping a lot

### DISCUSSION THREAD

**Discussion by de-Techneut** [blakeblackshear/frigate#8342] [Ask A Question]
When I go to debug mode, and check the bounding box, I can see they are jumping around the same object a lot. Which causes the cameras to create a lot of events. 

See the link below for an example: 
https://imgur.com/a/yF6waxc

Is there something I can do to improve this? So that the bounding box around stationary objects stay stationary?

**Comment by NickM-27** [comment]
I am assuming you are on frigate 0.13, there have been many improvements to the object tracking, stationary object handling, and many other things that will be coming in 0.13 beta3 that should improve situations like this

**Reply by de-Techneut** [reply]
Sorry, forgot to mention the version Frigate.  I'm indeed on frigate 13 beta 2. 

Thanks for the reply. Looking forward for beta 3 then.
